docs/extensions/api.md

```diff
@@ -172,37 +172,34 @@ def test_block_processor():
 Start with this example input:
 
 ``` text
-You could create zombies by mixing lime and coconut.
+A regular paragraph of text.
 
 !!!!!
-Never do that!
+First paragraph of wrapped text.
 
-Everyone might **die**!
+Second Paragraph of **wrapped** text.
 !!!!!
 
-Let's not.
+Another regular paragraph of text.
 ```
 
 The fenced text adds one node with two children to the tree:
 
-* **div**, with a `style` attribute.  It renders as 
+* `div`, with a `style` attribute.  It renders as 
   `<div style="display: inline-block; border: 1px solid red;">...</div>`
-    * **p** with text `Never do that!`
-    * **p** with text `Everyone might **die**`.  The conversion to a `<strong>` tag will happen when running the
-      inline processors, which is after all the block processors.
+    * `p` with text `First paragraph of wrapped text.`
+    * `p` with text `Second Paragraph of **wrapped** text`.  The conversion to a `<strong>` tag will happen when
+      running the inline processors, which will happen after all of the block processors have completed.
 
-The example display:
+The example output might display as follows:
 
-<hr>
-
-<p>You could create zombies by mixing lime and coconut.</p>
-<div style="display: inline-block; border: 1px solid red;">
-<p>Never do that!</p>
-<p>Everyone might <strong>die</strong>!</p>
-</div>
-<p>Let's not.</p>
-
-<hr>
+!!! note ""
+    <p>A regular paragraph of text.</p>
+    <div style="display: inline-block; border: 1px solid red;">
+    <p>First paragraph of wrapped text.</p>
+    <p>Second Paragraph of **wrapped** text.</p>
+    </div>
+    <p>Another regular paragraph of text.</p>
```

Comment:
Ideally these would be in blockquotes, but the theme we are using does not define any styles for blockquotes. Therefore, I wrapped them in `note` admonitions which have a blank label. The styling works out quite nicely:

![Screen Shot 2020-05-07 at 1 48 28 PM](https://user-images.githubusercontent.com/78846/81330665-68f65d80-906e-11ea-8fa5-c9d5b0b80d50.png)

Reply:
And I just realized I commented on a different example than the one I screenshot. Anyway, the point remains.